src/main/java/org/broadinstitute/hellbender/tools/picard/illumina/parser/ReadData.java

```diff
@@ -52,17 +41,6 @@ public void setQualities(final byte[] qualities) {
         this.qualities = qualities;
     }
 
-    /**
-     * @return Raw intensity values as produced by Illumina software, converted to shorts.
-     */
-    public FourChannelIntensityData getRawIntensities() {
-        return rawIntensities;
-    }
-
-    public void setRawIntensities(final FourChannelIntensityData rawIntensities) {
-        this.rawIntensities = rawIntensities;
-    }
```

Comment:
Again, removing random getters/setters like this just makes these classes less usable and consistent.